src/status_im/contexts/settings/wallet/events.cljs

```diff
@@ -53,3 +54,86 @@
      handle-connection)))
 
 (rf/reg-event-fx :wallet/get-key-pair-export-connection get-key-pair-export-connection)
+
+(defn extract-keypair-name
+  [db key-uids-set]
+  (when (= (count key-uids-set) 1)
+    (let [key-uid  (first key-uids-set)
+          keypairs (get-in db [:wallet :keypairs])]
+      (->> (filter #(= (:key-uid %) key-uid) keypairs)
+           first
+           :name))))
```

Comment:
Why do we only extract the key-pair name when there's one key-uid?

Reply:
It's because of design, when there is one key pair in toast we should display its name, but when it's more than one we just display the count in toast